e2e/js/js-cdn/scripts/copy-sdk.js

```diff
@@ -0,0 +1,14 @@
+// copy-sdk.js
+const fs = require('fs-extra');
+const path = require('path');
+
+const sdkSource = path.join(__dirname, '../../../../dist/sdk/js/devcycle.min.js');
+const appDestination = path.join(__dirname, '../../../../e2e/js/js-cdn/src/assets/devcycle.min.js');
```

Comment:
couldn't this path just go up one folder instead of return to root and go back down?

Reply:
so right